### PR TITLE
Cleanups related to rimraf and mkdirp

### PR DESCRIPTION
**Pull request by glandium**
Cf. https://github.com/taskcluster/docker-worker/pull/363#issuecomment-359734466 this might actually fix bug 1427096 too, obsoleting PR #363.

### CODE REVIEW

File: src/lib/volume_cache.js [hunk 1]
```diff
@@ -2,32 +2,13 @@ var debug = require('debug')('docker-worker:volumeCacheManager');
 var path = require('path');
 var Promise = require('promise');
 var fs = require('fs');
-var mkdirp = require('mkdirp');
-var rmrf = require('rimraf');
+const {makeDir, removeDir} = require('util/fs');
```
Comment (walac) [inline src/lib/volume_cache.js]: iirc, the correct syntax for importing the local module is `require('./util/fs')`

File: src/lib/features/dind.js [hunk 2]
```diff
@@ -35,11 +34,7 @@ class DockerInDocker {
     debug('image verified %s', imageId);
 
     // Create temporary directory
-    await new Promise((accept, reject) => {
-      fs.mkdir(this.tmpFolder, (err) => {
-        return err ? reject(err) : accept();
-      });
-    });
+    await makeDir(self.tmpFolder);
```
Comment (walac) [inline src/lib/features/dind.js]: \<google\>Did you mean `this.tmpFolder`?\</google\>

**Comment by glandium** [comment]
There are still failures, and in the case of the volume_cache ones, I really don't get why they happen as the changes are replacing functions with references to the exact same functions in another module.

**Comment by walac** [comment]
@glandium no worries, I am digging into it.

**Pull request merged**